### PR TITLE
Add PivManagementKeyPopup

### PR DESCRIPTION
**Pull request by emlun**
Example usage:

```
views.pivGetManagementKeyAndThen(
    function(pin) {
        console.log("pin", pin)
    },
    function(key) {
        console.log("key", key)
    }
);
```

### CODE REVIEW

File: ykman-gui/qml/ContentStack.qml [hunk 1]
```diff
@@ -125,6 +125,14 @@ StackView {
         replaceAtDepth(3, pivChangeManagementKeyView, 'pivChangeManagementKeyView')
     }
 
+    function pivGetManagementKeyAndThen(pinCallback, keyCallback) {
```
Comment (dagheyman) [inline ykman-gui/qml/ContentStack.qml]: Could we rename it to `pivGetPinOrManagementKey` ?

File: ykman-gui/qml/PivManagementKeyPopup.qml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import QtQuick 2.9
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.2
+import "utils.js" as Utils
```
Comment (dagheyman) [inline ykman-gui/qml/PivManagementKeyPopup.qml]: Utils is not used.

File: ykman-gui/qml/ContentStack.qml (hunk 1 above)
Comment (emlun) [reply ykman-gui/qml/ContentStack.qml]: Sure!

File: ykman-gui/qml/PivManagementKeyPopup.qml (hunk 2 above)
Comment (emlun) [reply ykman-gui/qml/PivManagementKeyPopup.qml]: Good catch, thanks!

**Pull request merged**